egs2/voices/asr1/conf/decode_asr.yaml

```diff
@@ -0,0 +1,3 @@
+beam_size: 60
```

Comment:
did not usually see 60 beam size, any specific reasons?

Reply:
I simply borrow this config from egs2/librispeech/asr1. The beam size seems to be updated by @sw005320.

Reply:
60 is too large.
20 is fine.